themes/default/content/docs/languages-sdks/javascript/_index.md

```diff
@@ -196,6 +196,91 @@ in the project root, or the environment variable `PULUMI_PREFER_YARN=true`,
 then Pulumi will use Yarn instead of NPM if the executable is available in the
 path.
 
+## Component Packages
```

Comment:
Should this be "Package Management with Components"? Or "Managing Package Version with Components"? I'm just wondering if there's a more descriptive/informative title we could use here.

Reply:
@scottslowe what about this:

> Understanding Dependencies on Provider Packages within Component Packages

Reply:
I'm fine with that (or just "Dependencies on..." without the "Understanding")